section2/exercises/methods.rb

```diff
@@ -12,19 +12,51 @@ def print_name
 
 # Write a method that takes a name as an argument and prints it:
 def print_name(name)
-  # YOUR CODE HERE
+  puts "Albus Dumbledore"
```

Comment:
🚨Hmmm... if you are passing `name`, should you statically print `Albus Dumbledore`?